### PR TITLE
Organize unit test assets

### PR DESCRIPTION
**Pull request by birkskyum**
Integration test suites are moved to integration test folder. Unit test utilities and assets are moved to unit test folder.

**Before**

test
- browser
- integration
-  - ... other suites
- expected ( >  integration/style-shaping )
- fixtures ( > unit/assets )
- util (> unit/lib)
- unit
- - style-spec
- - - fixtures

**After**

test
- unit
-  -  lib
-  -  assets
- integration
-  - browser
-  - style-spec (new git workflow + `npm run test-style-spec`)
-  - symbol-shaping  (new git workflow + `npm run test-symbol-shaping`)
-  - ... other suites

### CODE REVIEW

**Comment by HarelM** [comment]
I would consider changing misc-assets to just assets.
And maybe change util to lib like it is in the integration, or change the lib to util in the integration. I think it should be the same as they provide the same idea...?

Also, I think the tests that has an "update" mechanism are not unit tests almost by definition, I think they should be elsewhere (I think it is related to the fixtures folders, right?)

Besides the above nitpicking, I think this PR is great, thanks!

**Comment by birkskyum** [comment]
> Also, I think the tests that has an "update" mechanism

Which tests here have an update? 

The way I sorted this currently, is that /test/unit contains all the assets that are imported from .test.ts files in the /src.

The remaining folders in /test all use the bundles.

**Comment by HarelM** [comment]
Validate-spec and shaping.
Look for the UPDATE variable to find those tests.
I think they are the tests that have the fixtures folders you renamed.

**Comment by birkskyum** [comment]
> Validate-spec and shaping. Look for the UPDATE variable to find those tests. I think they are the tests that have the fixtures folders you renamed.

Those tests are currently a part of the `test-unit` suite, so do you think we should move them to a different suite as well?

**Comment by HarelM** [comment]
I think they should move to another suite, yes.
It will clean the folder structure as well.
It feels to me like integration tests, so it might be a good fit to place them there.

**Comment by birkskyum** [comment]
I have moved "browser" "style-spec" and "symbol-shaping" folders into "integration", and added new workflows for the two latter

File: .github/workflows/test-style-spec.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+name: Inegration Test - Style Spec
```
Comment (HarelM) [inline .github/workflows/test-style-spec.yml:1]: Typo.
Also if you prefix these too you should prefix the others as well (render, query, etc)

Comment (birkskyum) [reply .github/workflows/test-style-spec.yml]: Renamed all of them for consistency.

**Review by HarelM — APPROVED** [review]
Amazing!! So much cleaner now!

**Pull request merged**